overviews/core/_posts/2012-09-20-futures.md

```diff
@@ -615,8 +615,30 @@ Invoking the `future` construct uses a global execution context to start an asyn
 Support for extending the Futures API with additional utility methods is planned.
 This will allow external frameworks to provide more specialized utilities.
 
+
 ## Blocking
 
+### in a Future
+
+As seen with the global `ExecutionContext`, it is possible to notify an `ExecutionContext` of a blocking call with the `blocking` construct.
+The implementation is however at the complete discretion of the `ExecutionContext`. While some `ExecutionContext` such as `ExecutionContext.global`
+implement `blocking` by means of `ManagedBlocker`, some just do nothing:
+
+    implicit val ec = ExecutionContext.fromExecutor(
+                        Executors.newFixedThreadPool(4))
+    Future {
+      // here blocking serves only for documentation purpose
+      blocking { blockingStuff() }
+    }
+
+Is equivalent to
```

Comment:
I think that would be confusing to say "can be". My point is that in this situation (ExecutionContext backed by an ExecutorService) the `blocking` block isn't doing anything. So from the JVM standpoint it _is_ equivalent. It's like saying: `(); 42` is equivalent to `42`.
Maybe I could write `Is equivalent at runtime to`? Or `has the same effect as`?